### ISSUE TITLE
Missing heading level in text only

### ISSUE THREAD

**Issue by laudrain** [daisy/ace#89]
When a heading level is missing, it is signaled in the ace.json file in the text of HTML "Headings outline" fort the report:
<span class=\"toc-missing\">Missing heading h3</span>

As it is poorly structured, this kind of error will be difficult to extract from the json...

**Comment by rdeltour** [comment]
> this kind of error will be difficult to extract from the json...

Yeah, the outlines are primarily used for manual inspection. Now, I can see how having them as structured JSON data could be useful.
